Mod0/TechnicalDay2.md

```diff
@@ -0,0 +1,38 @@
+# How Humans Keep Track of Information and What That Means for Programming
+
+Like yesterday, this is an introduction designed to get you looking at code, getting familiar with syntax, and starting to think like a developer.  The expectation is that you will NOT master any topics today!
+
+### What is a Thing?
+
+As humans, we often interact with physical objects in the world; let's call them `things`.  We use ourknowledge and experience to work with many different types of things with no issue.  We know to sit on a chair, and lay on a bed; even though these things are both furniture, we know that they have different uses.
+
+We also use non-tangible objects like vocabulary, numbers, and dates which allow us to communicate with each other; let's call these objects `data`.  We can use data to communicate our needs, perform math and statistics, and relate spans of time (and many other tasks!). Using data, we can understand sentences like: On Tuesday, my son will be 12 years old. He is getting 2 lego sets for his birthday.
+
+With objects and data, we have everything we need to accomplish our daily tasks.  But there is one more concept that humans rely on: lists.  We use lists to help organize all of the objects and data we know about.  Let's imagine we are moving into a new home.  We know that our clothes and furniture need to be moved.  From this, we can infer that multiple pieces of clothing and multiple pieces of furniture exist and must be moved; we use the idea of `lists` instead of explicitly stating every single item that needs to be moved.
```

Comment:
The first two paragraphs about things and data are great! I think they quickly move forward the ideas. 

This one about lists feels more confusing to me. I'm not following the clothing and furnature example, I think because in real world I don't think I'd use a list there I'd just throw it all in a box 😆  And the first two sentences maybe because they are so broad feel a bit confusing too. 

Maybe starting with something like the grocery store example and introducing the helpful additional tool of a list might be smoother?

Reply:
I agree - I think maybe I overthought how much we really _need_ to introduce lists as a concept.